dev/archery/archery/integration/runner.py

```diff
@@ -158,7 +158,6 @@ def _gold_tests(self, gold_dir):
                 skip_testers.add("JS")
                 skip_testers.add("Rust")
             if prefix == '2.0.0-compression':
-                skip_testers.add("C#")
```

Comment:
I've enabled the compression integration tests here, but it looks like those only test reading compressed data, so could have been enabled previously. As far as I can see the integration tests aren't currently set up to allow reading compressed IPC data written by C# from another implementation.